qr_generator/lib/repositories/seed_repository.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:math';
+
+import 'package:flutter/foundation.dart';
+import 'package:qr_generator/model/seed_model.dart';
+import 'package:qr_generator/utils/api_client.dart';
+
+class SeedRepository {
+  final ApiClient netUtil;
+  SeedRepository({@required this.netUtil}) : assert(netUtil != null);
+
+  //Get new seed data 
+  Future<SeedModel> getSeedData() async {
+    const url = 'seed';
```

Comment:
**suggestion:** define constant "urls" public to the file so that you could access them in tests or other areas as well.

Otherwise,
**suggestion:** no need to define the const if only used once in next line, it'd likely read better.

Reply:
Done!